rai/client.go

```diff
@@ -338,34 +338,31 @@ func unmarshal(rsp *http.Response, result interface{}) error {
 
 // readArrowFiles read arrow files content and returns a list of ArrowRelations
 func readArrowFiles(files []TransactionAsyncFile) ([]ArrowRelation, error) {
-	res := make(map[string][][]interface{})
+	var out []ArrowRelation
 	for _, file := range files {
 		if file.ContentType == "application/vnd.apache.arrow.stream" {
 			reader, err := ipc.NewReader(bytes.NewReader(file.Data))
 			if err != nil {
-				return nil, err
+				return out, err
```

Comment:
I put this back to what it was before - but i'm not sure if maybe returning `nil` was actually better?

@NRHelmi can you say what to do here?

Reply:
I think it's better to keep it, avoiding null exceptions :D